src/schemas/json/tslint.json

```
@@ -1818,6 +1818,23 @@
             }
           ]
         },
+        "no-for-in": {
+          "description": "Ban the usage of for…in statements.",
```

```suggestion
          "description": "Forbid for…in statements\nhttps://palantir.github.io/tslint/rules/no-for-in/",
```

Optional fix: description can be simplified.
Optional fix: link to the documentation can be added.